crates/red_knot_test/src/diagnostic.rs

```diff
@@ -3,9 +3,10 @@
 //! We don't assume that we will get the diagnostics in source order.
 
 use ruff_source_file::{LineIndex, OneIndexed};
-use ruff_text_size::Ranged;
```

Comment:
It's not a problem, but I'm curious why we stopped using the `Ranged` trait here; the reason isn't obvious to me.

Reply:
Oh, I should have reverted that change. It's because `Diagnostic` returns `Option<TextRange>` which wasn't compatible. Anyway, I went ahead and deleted the `mdtest` Diagnostic and replaced it with `Diagnostic`.